src/Store/routes.ts

```diff
@@ -1,63 +1,56 @@
 import { lazy } from "solid-js";
 import { TNavigation } from "./side-bar-store";
 
-const Docs = lazy(() => import("../Pages/Docs/index"));
-const Home = lazy(() => import("../Pages/Home/index"));
-const Playground = lazy(() => import("../Pages/Playground/index"));
-const NotFound = lazy(() => import("../Pages/NotFound/index"));
+const NotFound = lazy(() => import("../Pages/NotFound"));
+
 
 export const routes: TNavigation[] = [
```

Comment:
this is an array that Signals are generated from, so I wonder whether we could import those instead of creating signals